xosc_dl.py

```diff
@@ -144,7 +145,13 @@ def __init__(self, test_mode=False):
         t.start()
 
     def about_dialog(self):
-        QMessageBox.about(self, f"About OSCDL",
+        QMessageBox.about(self, f"About OSCDL","""<style>        
+
+        a{  
+            color:#0F8EC2;
+            text-decoration: none;
+        }        
+    </style>"""
```

Comment:
The styling here should probably be a part of the stylesheet.

Reply:
That was my original intention, I am just not sure it's doable

I did not find any ways to edit a QMessageBox html text through the stylesheet. I even found some forum posts specifically claiming you can't change link colors in qt MessageBoxes, so any solution must be hard-coded. I'll look into it further.

Plus, I haven't found any other links anywhere one the app, so it's not like we will be saving any time by placing the styling on the stylesheet (although I get it on principle)

Reply:
**Update:** I am currently omitting this from the changes as you can see in the newest commit. I'll try to create a solution that is theme specific, but I have read from several sources that QLabel and QMessageBox links are not customizable through stylesheets. Would you prefer an app-wide solution  (people online prefer customizing the QPalette), or just keep the original dark blue and somewhat hinder readability on one page for the dark theme?

Reply:
Might have to keep the dark blue here for now.
There's another hyperlink in the program in the main window, you can find it by clicking on the icon next to the author name when selecting an app.